Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/server/BaseService.java

```diff
@@ -65,25 +75,46 @@ protected AssertionType getAssertion(WebServiceContext context, AssertionType as
             assertion = assertionIn;
         }
 
-        // Extract the message id value from the WS-Addressing Header and place it in the Assertion Class
         if (assertion != null) {
```

Comment:
If the context is null the assertion will be null...or if the assertion can't be extracted.